### DISCUSSION TITLE
Support for Admob's "App open" ad unit?

### DISCUSSION THREAD

**Discussion by DrOwlDev** [marcojak/MauiMTAdmob#62] [Ideas]
any chance to add that support?

**Comment by marcojak** [comment]
Hi.
Yes, I'm currently working on this.
The Android version already works but I need to complete it adding the required events and configurations.
I'll then see if I can implement the same on iOS.
I hope to release it between the end of this week and mid of the next one.

**Reply by DrOwlDev** [reply]
thanks! looking forward to see the Andriod version in action as I dont have a Mac to build IOS apps haha.
Will be super great to also update your sample code showing how it works. 
many thanks

**Reply by DrOwlDev** [reply]
Also, any chance to show a sample implementation of "Native advanced" ad unit? is it very complex? This is low priority I suppose ~

**Comment by marcojak** [comment]
Hi.
Just a quick update: Open ads are working fine both on Android and iOS.
I now just have to clean and improve the code then update the documentation and finally, I'll release the new version (I guess within the next few days).
While I'm working on the new version, I'm also updating the sample (this is how I test if everything works as expected) so it should be available soon.

For the Native Ads, they are more complicated as they involve the use of custom native views to add the ads to. After finishing on the Open Ads, I'll see how I can add the Native ads too.

**Comment by marcojak** [comment]
Version 1.2.0 with App Open Ads is online and the sample code has been updated to support it

**Reply by DrOwlDev** [reply]
oh great! let me try it out! thanks
